src/annotator/sidebar.js

```diff
@@ -63,54 +63,50 @@ export default class Sidebar extends Guest {
   constructor(element, config) {
     super(element, config);
 
-    let externalFrame;
-    let frame;
-    let hypothesisSidebar; // refers to <hypothesis-sidebar> element
-
-    if (config.externalContainerSelector) {
-      externalFrame =
-        document.querySelector(config.externalContainerSelector) || element;
+    this.iframe = createSidebarIframe(config);
+
+    const {
+      externalContainerSelector,
+      theme,
+      openSidebar,
+      annotations,
+      query,
+      group,
+    } = config;
+
+    if (externalContainerSelector) {
+      this.externalFrame =
+        /** @type {HTMLElement} */
+        (document.querySelector(externalContainerSelector)) ?? element;
+      this.externalFrame.appendChild(this.iframe);
     } else {
-      frame = document.createElement('div');
-      frame.style.display = 'none';
-      frame.className = 'annotator-frame';
+      this.iframeContainer = document.createElement('div');
```

Comment:
This is definitely a better name than `frame`.

Reply:
Thanks. The first time I took a look at the class I found confusing the `this.frame` naming.